### ISSUE TITLE
Easy PSI 中Kuscia Api的使用

### ISSUE THREAD

**Issue by T-ze-yu** [secretflow/kuscia#189]
在Easy PSI中原来的API端口8082映射到了8081，通过拿到容器内的验证token和证书等，可以在另外的机器用curl命令来使用Kuscia Api。
其中使用了如下命令来创建job：
curl -X POST 'https://192.168.201.164:8081/api/v1/job/create'  --cert 'certificate/kusciaapi-client.crt'  --key 'certificate/kusciaapi-client.key'  --cacert 'certificate/ca.crt' --header "Token: $(cat certificate/token)" --header 'Content-Type: application/json' -k -d '{
         "job_id": "xx",
         "initiator": "alice",
         "max_parallelism": 2,
         "tasks": [{
                 "app_image": "psi-image",
                 "parties": [{"domain_id": "alice"},{"domain_id": "bob"}],
                 "alias": "job-psixx",
                 "task_id": "job-psixx",
                 "task_input_config": "{\"sf_datasource_config\":{\"alice\":{\"id\":\"default-data-source\"},\"bob\":{\"id\":\"default-data-source\"}},\"sf_cluster_desc\":{\"parties\":[\"alice\",\"bob\"],\"devices\":[{\"name\":\"spu\",\"type\":\"spu\",\"parties\":[\"alice\",\"bob\"],\"config\":\"{\\\"runtime_config\\\":{\\\"protocol\\\":\\\"REF2K\\\",\\\"field\\\":\\\"FM64\\\"},\\\"link_desc\\\":{\\\"connect_retry_times\\\":60,\\\"connect_retry_interval_ms\\\":1000,\\\"brpc_channel_protocol\\\":\\\"http\\\",\\\"brpc_channel_connection_type\\\":\\\"pooled\\\",\\\"recv_timeout_ms\\\":1200000,\\\"http_timeout_ms\\\":1200000}}\"},{\"name\":\"heu\",\"type\":\"heu\",\"parties\":[\"alice\",\"bob\"],\"config\":\"{\\\"mode\\\": \\\"PHEU\\\", \\\"schema\\\": \\\"paillier\\\", \\\"key_size\\\": 2048}\"}],\"ray_fed_config\":{\"cross_silo_comm_backend\":\"brpc_link\"}},\"sf_node_eval_param\":{\"domain\":\"preprocessing\",\"name\":\"psi\",\"version\":\"0.0.1\",\"attr_paths\":[\"input/receiver_input/key\",\"input/sender_input/key\",\"protocol\",\"precheck_input\",\"bucket_size\",\"curve_type\"],\"attrs\":[{\"ss\":[\"id1\"]},{\"ss\":[\"id2\"]},{\"s\":\"ECDH_PSI_2PC\"},{\"b\":true},{\"i64\":\"1048576\"},{\"s\":\"CURVE_FOURQ\"}]},\"sf_input_ids\":[\"alice-table\",\"bob-table\"],\"sf_output_ids\":[\"psi-output\"],\"sf_output_uris\":[\"psi-output.csv\"]}",
                 "priority": "100"
             }, {
                 "app_image": "psi-image",
                 "parties": [{"domain_id": "alice"},{"domain_id": "bob"}],
                 "alias": "job-splitxx",
                 "task_id": "job-splitxx",
                 "dependencies": ["job-psixx"],
                 "task_input_config": "{\"sf_datasource_config\":{\"alice\":{\"id\":\"default-data-source\"},\"bob\":{\"id\":\"default-data-source\"}},\"sf_cluster_desc\":{\"parties\":[\"alice\",\"bob\"],\"devices\":[{\"name\":\"spu\",\"type\":\"spu\",\"parties\":[\"alice\",\"bob\"],\"config\":\"{\\\"runtime_config\\\":{\\\"protocol\\\":\\\"REF2K\\\",\\\"field\\\":\\\"FM64\\\"},\\\"link_desc\\\":{\\\"connect_retry_times\\\":60,\\\"connect_retry_interval_ms\\\":1000,\\\"brpc_channel_protocol\\\":\\\"http\\\",\\\"brpc_channel_connection_type\\\":\\\"pooled\\\",\\\"recv_timeout_ms\\\":1200000,\\\"http_timeout_ms\\\":1200000}}\"},{\"name\":\"heu\",\"type\":\"heu\",\"parties\":[\"alice\",\"bob\"],\"config\":\"{\\\"mode\\\": \\\"PHEU\\\", \\\"schema\\\": \\\"paillier\\\", \\\"key_size\\\": 2048}\"}],\"ray_fed_config\":{\"cross_silo_comm_backend\":\"brpc_link\"}},\"sf_node_eval_param\":{\"domain\":\"preprocessing\",\"name\":\"train_test_split\",\"version\":\"0.0.1\",\"attr_paths\":[\"train_size\",\"test_size\",\"random_state\",\"shuffle\"],\"attrs\":[{\"f\":0.75},{\"f\":0.25},{\"i64\":1234},{\"b\":true}]},\"sf_output_uris\":[\"train-dataset.csv\",\"test-dataset.csv\"],\"sf_output_ids\":[\"train-dataset\",\"test-dataset\"],\"sf_input_ids\":[\"psi-output\"]}",
                 "priority": "100"
             }
         ]
     }' -i
返回结果如下：
{"status":{"code":0,"message":"success","details":[]},"data":{"job_id":"xx"}}
通过查询接口如下命令进行查询：
curl -X POST 'https://192.168.201.164:8081/api/v1/job/status/batchQuery'  --cert 'certificate/kusciaapi-client.crt'  --key 'certificate/kusciaapi-client.key'  --cacert 'certificate/ca.crt' --header "Token: $(cat certificate/token)" --header 'Content-Type: application/json' -k -d '{
    "job_ids": ["xx"]
}' -i
返回结果如下：
{"status":{"code":0,"message":"success","details":[]},"data":{"jobs":[{"job_id":"xx","status":{"state":"Failed","err_msg":"","create_time":"2023-12-08T06:25:21Z","start_time":"2023-12-08T06:25:21Z","end_time":"2023-12-08T06:25:25Z","tasks":[{"task_id":"job-psixx","state":"Failed","err_msg":"The remaining non-failed parties counts 1 is less than the task success threshold 2","create_time":"2023-12-08T06:25:21Z","start_time":"2023-12-08T06:25:21Z","end_time":"2023-12-08T06:25:25Z","parties":[{"domain_id":"alice","state":"Failed","err_msg":""},{"domain_id":"bob","state":"Failed","err_msg":"container[secretflow] terminated state reason \"Error\", message: \"terminate called after throwing an instance of 'yacl::EnforceNotMet'\\n  what():  [Enforce fail at psi/psi/kuscia_adapter.cc:60] google::protobuf::util::JsonStringToMessage( task_input_config_str, &task_input_config) .ok(). \\nStacktrace:\\n#0 main+0x976635\\n#1 __libc_start_main+0x7f564e23acf3\\n\\n\""}]},{"task_id":"job-splitxx","state":"","err_msg":"","create_time":"","start_time":"","end_time":"","parties":[]}]}}]}}
可以看到任务并未成功运行，帮忙看看原因。
